packages/text/src/react/list/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { compose, css } from 'glamor'
+import React, {HTMLAttributes} from 'react'
+
+import { useTheme, names } from '@pluralsight/ps-design-system-theme'
+
+import ListItem from './list-item'
+
+import stylesheet from '../../css'
+import * as vars from '../../vars'
+
+const styles = ({ themeName, type}: { themeName: keyof typeof names; type: keyof typeof vars.listTypes}) =>
+    compose(
+      css(stylesheet[`.psds-text__list.psds-theme--${themeName}`]),
+      css(stylesheet[`.psds-text__list`]),
+      css(stylesheet[`.psds-text__list--type-${type}`])
+    )
+
+interface ListStatics {
+  types: typeof vars.listTypes
+  Item: typeof ListItem
+}
+    
+interface ListProps extends HTMLAttributes<HTMLUListElement | HTMLOListElement> {
+  type?:  keyof typeof vars.listTypes
```

Comment:
same ValueOf

Reply:
Just realized this is exactly the same when I went back to look at the type. It also does not shorten it but I'll use it.